src/Router.ts

```diff
@@ -669,6 +831,24 @@ class RouterImpl {
 		// Return the key value pairs.
 		return pathParams;
 	}
+
+	/**
+	 * Get the full path and URL parameters string for a given path, removing the protocol, server, and port information.
+	 * 
+	 * @param source - The URL to inspect.
+	 * 
+	 * @returns The path string.
+	 */
+	private _getPathString(source: string): string {
+
+		const result = source.replace(window.location.origin, '');
+
+		if (!result || result === '') {
+			return '/';
```

Comment:
This could have potential breaking changes, perhaps this needs to configure with a backwards compatible default value?

Reply:
Reverted to align with original